src/Compute/Compute/Generated/DiskEncryptionSet/Config/NewAzureRmDiskEncryptionSetConfigCommand.cs

```diff
@@ -51,8 +51,9 @@ public partial class NewAzureRmDiskEncryptionSetConfigCommand : Microsoft.Azure.
         [Parameter(
             Mandatory = false,
             Position = 2,
-            ValueFromPipelineByPropertyName = true)]
-        [PSArgumentCompleter("SystemAssigned")]
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The type of Managed Identity used by the DiskEncryptionSet. Only SystemAssigned is supported for new creations. Disk Encryption Sets can be updated with Identity type None during migration of subscription to a new Azure Active Directory tenant; it will cause the encrypted resources to lose access to the keys.")]
+        [PSArgumentCompleter("SystemAssigned", "UserAssigned", "SystemAssigned, UserAssigned")]
```

Comment:
Hi @grizzlytheodore , please follow the best practice for managed identity to design your new parameter, see https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/managed-identity-best-practices.md#new--cmdlet-design-practices